localstack/services/sqs/models.py

```diff
@@ -249,13 +250,18 @@ def url(self, context: RequestContext) -> str:
             # or us-east-2.queue.localhost.localstack.cloud:4566/000000000000/my-queue
             region = "" if self.region == "us-east-1" else self.region + "."
             scheme = context.request.scheme
-            host_url = f"{scheme}://{region}queue.{constants.LOCALHOST_HOSTNAME}:{config.EDGE_PORT}"
+
+            host_definition = localstack_host(use_localhost_cloud=True)
+            host_url = f"{scheme}://{region}queue.{host_definition.host_and_port()}"
         elif config.SQS_ENDPOINT_STRATEGY == "path":
             # https?://localhost:4566/queue/us-east-1/00000000000/my-queue (us-east-1)
             host_url = f"{context.request.host_url}queue/{self.region}"
         else:
             if config.SQS_PORT_EXTERNAL:
-                host_url = external_service_url("sqs")
+                host_definition = localstack_host(
+                    use_hostname_external=True, custom_port=config.SQS_PORT_EXTERNAL
+                )
+                host_url = f"{get_protocol()}://{host_definition.host_and_port()}"
```

Comment:
hey! i can see that `SQS_PORT_EXTERNAL` is still in the control path here although we decided to boot it. can we just remove this?

Reply:
This PR is just matching existing behaviour. I'll get rid of it in a follow up PR targeting the `v2` branch once this is merged :+1: